cedar-policy/src/api.rs

```diff
@@ -1530,17 +1530,30 @@ impl EntityUid {
     }
 }
 
-// This FromStr implementation requires the _normalized_ representation of the
-// UID. See https://github.com/cedar-policy/rfcs/pull/9/.
 impl FromStr for EntityUid {
     type Err = ParseErrors;
 
-    /// This is deprecated (starting with Cedar 1.2); use
-    /// `EntityUid::from_type_name_and_id()` or `EntityUid::from_json()`
-    /// instead.
-    //
-    // You can't actually `#[deprecated]` a trait implementation or trait
-    // method.
+    /// Parse an [`EntityUid`].
+    ///
+    /// An [`EntityUid`] consists of a sequence of [`EntityTypeName`]s joined by `::`,
```

Comment:
this is not right.  The `EntityTypeName` is the type name with all namespaces.  There's exactly one `EntityTypeName` in any `EntityUid`.

Reply:
Nice catch, thanks